Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,34 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "https://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "https://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "https://anongit.freedesktop.org/git/gstreamer/qt-gstreamer.git"
+
+  depends_on "boost"
+  depends_on "cmake" => :build
+  depends_on "glib"
+  depends_on "gstreamer"
+  depends_on "gst-plugins-base"
+  depends_on "pkg-config" => :build
+  depends_on "qt5"
+
+  def install
+    # https://bugzilla.gnome.org/show_bug.cgi?id=746760
+    # https://bugzilla.gnome.org/show_bug.cgi?id=733788
+    # https://bugzilla.gnome.org/show_bug.cgi?id=763201
+    rm "#{buildpath}/src/QGst/Memory"
+
+    # QtGStreamer will NOT build on a case-insensitive HFS+ filesystem.
+    # So create a sparse image formatted as HFS+ case sensitive and build it there.
+    system "cmake", "-DQT_VERSION=5", "-DCMAKE_INSTALL_PREFIX=#{prefix}", "-DCMAKE_CXX_FLAGS=-I#{HOMEBREW_PREFIX}/lib/gstreamer-1.0/include"
```

Comment:
Instead of using `HOMEBREW_PREFIX`, try fetching this from its own prefix: `#{Formula["gstreamer"].opt_lib}/gstreamer-1.0/include`